### DISCUSSION TITLE
Finder pattern in specific colour

### DISCUSSION THREAD

**Discussion by Paulo--M--** [chillerlan/php-qrcode#232] [Q&A] [answered]
Hello,
is it possible to generate a PNG QR code with the finder pattern in a specific colour?
Thanks in advance for any solutions/pointers.
Paulo

**Comment by codemasher** [comment]
Yes, that's demonstrated in several examples: [GD example](https://github.com/chillerlan/php-qrcode/blob/main/examples/image.php), [Imagick example](https://github.com/chillerlan/php-qrcode/blob/main/examples/imagick.php), etc...

**Comment by Paulo--M--** [comment, marked as answer]
Thank you -- the pointer was much appreciated. :-)
